UltiSnips/nix.snippets

```diff
@@ -0,0 +1,38 @@
+snippet sha256 "sha256"
+sha256 = "0000000000000000000000000000000000000000000000000000";
```

Comment:
that one I find the most useful one.

Reply:
There's also `pkgs.lib.fakeSha256`

Reply:
This one is more universal since you cannot expect `pkgs` always to be in scope.

Reply:
Fair enough :sweat_smile: